src/clanhr/ring_test_client/core.clj

```diff
@@ -5,6 +5,12 @@
   (:use clojure.test
         ring.mock.request))
 
+(defn ->edn [raw]
```

Comment:
You can read the names book by Zach Tallman. He actually says that we should always use `->edn` to convert stuff and that something names `string->int` should be a map like `{"a" 1}`.